docs/guides/global-concurrency-limits.md

```diff
@@ -15,6 +15,8 @@ Rate Limits ensure system stability by governing the frequency of requests or op
 
 When selecting between Concurrency and Rate Limits, consider your primary goal. Choose Concurrency Limits for resource optimization and task management. Choose Rate Limits to maintain system stability and fair access to services.
 
+The core difference between a rate limit and a concurrency limit is the way in which slots are released. With a rate limit, slots are released at a controlled rate, controlled by `slot_decay_per_second` whereas with a concurrency limit, slots are released when the concurrency manager is exited. 
```

Comment:
Can we add a bit more about how slot decay works? If I set it to 10, are my slots released after 10 seconds?

Reply:
`slot_decay_per_second` is the rate at which `n` slots decay in a second. So if you set it to `10`, then 10 slots will become available again every second. There is already a section on slot decay, I'll try to make it a bit more detailed.

Reply:
https://github.com/PrefectHQ/prefect/pull/10886/commits/62cd88f20545240f72be81d1518eb82444676a64

Reply:
Thank you!